interface/ceed-operator.c

```diff
@@ -1541,7 +1559,16 @@ int CeedOperatorGetContextFieldLabel(CeedOperator op, const char *field_name, Ce
 
     return CEED_ERROR_SUCCESS;
   } else {
-    return CeedQFunctionContextGetFieldLabel(op->qf->ctx, field_name, field_label);
+    CeedCall(CeedQFunctionContextGetFieldLabel(op->qf->ctx, field_name, field_label));
+    if (*field_label) {
+      CeedInt num_operators = (*field_label)->num_operators + 1;
+
+      if (num_operators == 1) CeedCall(CeedCalloc(num_operators, &(*field_label)->op));
```

Comment:
nit-pick, but
```suggestion
      if ((*field_label)->num_operators == 0) CeedCall(CeedCalloc(num_operators, &(*field_label)->op));
```

Would work right? Seems a bit more straight forward to me.

Reply:
six of one, half dozen of the other since `num_operators = (*field_label)->num_operators`